packages/manager/apps/telecom/src/components/uib-dropdown-helper/autoscroll-on-toggle.provider.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+
+/*
+ * uib-dropdown helper to center the viewport on the dropdown when opened
+ */
+export default function autoScrollOnToggleProvider() {
+  this.$get = /* @ngInject */ ($timeout) => {
```

Comment:
**nitpick**
Is there a reason you declare it as a provider ? instead of a factory or service ?

Reply:
i didn't choose "service" because we are using services for business/model related methods and this one is just a dirty trick to center the view on uib-dropdowns